api/src/pcapi/settings.py

```diff
@@ -95,6 +95,7 @@
 
 
 # SENTRY
+SENTRY_ENABLED = bool(int(os.environ.get("SENTRY_ENABLED", 1)))
```

Comment:
Les `settings` existants sont de la forme `ENABLE_*`, aucun n'est en `*_ENABLED`, peut-on essayer de garder une certaine homogénéité ?